apps/form-builder/src/app/api/forms/[formId]/formSubmissions/route.ts

```diff
@@ -0,0 +1,12 @@
+// function to get all form submisions
+import { getFormSubmissions } from "@/server/actions/formSubmissions";
+import { NextRequest, NextResponse } from "next/server";
+import { zObjectId } from '@hack4impact-utk/internal-models';
+
+export async function GET (_request: NextRequest, { params }: { params: { formId: string } }) {
+    // if formId is valid, get form submissions and return them with status code 200
+    if(zObjectId.safeParse(params.formId).success) {
+        const formSubmissions = getFormSubmissions(params.formId)
+        return NextResponse.json(formSubmissions, { status: 200 })
```

Comment:
This is on me for not specifying this in the story description, but if `formSubmissions` is null, that indicates that we couldn't find the form in the database. In this instance, we should return a response with a status code of 404 and a message indicating that we couldn't find the form. [Here ](https://github.com/hack4impact-utk/internal-applications/blob/main/apps/form-builder/src/app/api/forms/%5BformId%5D/route.ts )is some existing code to reference, lines 17-19